.github/container/bump.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+## Parse command-line arguments
+
+usage() {
+cat <<EOF
+Usage: $0 [OPTION]...
```

Comment:
[minor] It would be nice to have a brief description what the purpose of the script is and what it does.

Reply:
Done